workshop/Lab2/README.md

```diff
@@ -0,0 +1,144 @@
+# Lab 2 Deploy an application with a manually provisioned backing store
+
+Learn to manually provision a Redis instance, and manually connect it to an
+application hosted on the IBM Kubernetes Service.
+
+# Deploy your application
+
+If you're continuing from Lab 1, you can skip this step. Otherwise, run
+
+```
+$ kubectl run guestbook --image=ibmcom/guestbook:v1
+$ kubectl expose deployment guestbook --type="NodePort" --port=3000
+```
+
+This will deploy Guestbook and expose a port so you can connect to it. Note that
+it is storing state locally in memory.
+
+# Manually provision a Redis instance
+
+We want some sort of stable backing store for our application to store it's state in. For the purposes
+of this workshop, we'll be using a service available publically on IBM Cloud. While it would technically
+be possible to plug this service directly into our Kube cluster using the IBM Cloud Dashboard, this step is
```

Comment:
I'm not following the point of the sentences starting with "While it would be...".  Can you elaborate on what the intent of these sentences are?

Reply:
I want to acknowledge that what I'm doing here is intentionally not the right way to do things - IBM Cloud lets you attach services to your cluster using the dashboard already, doesn't it? Using the thing that lets you share services between CF, Kube, and whatever?

Reply:
What if it said something like this instead for everything in this paragraph starting with "While it would...":
We could technically use any Redis service that is accessible from your Kubernetes cluster.

Reply:
hmmm I don't see Redis any more in the catalog - at least not a free version. We may need to deploy our own.